src/utils/platform.rs

```diff
@@ -1,22 +1,25 @@
 use std;
 
-#[cfg(not(target_os = "macos"))]
+#[cfg(not(any(target_os = "macos", target_os = "linux")))]
 pub fn get_block_size() -> u64 {
     1024
 }
 
-#[cfg(target_os = "macos")]
+#[cfg(any(target_os = "macos", target_os = "linux"))]
 pub fn get_block_size() -> u64 {
     512
 }
```

Comment:
I've discovered its a bit weirder than that. I'll have a commit to fix up the get_block_size() soon.